src/main/webapp/community.html

```diff
@@ -12,9 +12,8 @@
         fetch(url).then((response) => {
           return response.json();
         }).then((users) => {
-          const userlist = document.getElementById('userlist');
+          const userlist = document.getElementById('list');
```

Comment:
I would go the other way and err on the side of more details. So I would update the element `id` to be `userlist` and leave the javascript part as it is.

Reply:
resolved!